lib/doorkeeper/config.rb

```diff
@@ -154,22 +154,53 @@ def enforce_content_type
         @config.instance_variable_set(:@enforce_content_type, true)
       end
 
+      # Configure the secret storing functionality
+      def configure_secrets_for(type, using:, fallback:)
```

Comment:
Can we make it a private method? Or there is some reason to be public?

Reply:
No reason, will do